headers/types/common/common.h

```diff
@@ -1072,6 +1072,517 @@ struct wavi_data {
+
+struct dse_driver_settings {
+    uint32_t error_flags;
+    uint32_t field_0x4;
+
+    void* heap_location;
+    uint32_t heap_size;
+
+    uint8_t field_0xC;
+    uint8_t is_tick_rate_60hz;
+    uint8_t field_0xE;
+    uint8_t field_0xF;
+    int16_t field_0x10;
+    int16_t field_0x12;
+    uint16_t num_se_sequences;
+    uint16_t tracks_per_se_sequence;
+
+    struct dse_heap_allocator heap_allocator;
+
+    void* error_callback;
+    void* error2_callback;
+    uint32_t field_0x30;
+};
+ASSERT_SIZE(struct dse_driver_settings, 52);
+
+struct dse_driver_work {
+    int8_t initialized;
+    uint8_t stopped;
+    uint16_t unknown_0x2;
+    int error_code;
+    uint32_t error_data;
+    uint32_t error_data2;
+    int error2_code;
+    uint32_t error2_data;
+    uint32_t error2_data2;
+    void* error_callback;
+    void* error2_callback;
+    uint32_t field_0x24;
+    int16_t microseconds_per_driver_tick;
+    uint16_t field_0x2A;
+    int total_driver_ticks;
+    int driver_tick_rate;
+    uint32_t current_rng_seed;
+    uint8_t field_0x38;
+    uint8_t field_0x39;
+    uint8_t last_se_bank_num_sequences;
+    uint8_t last_se_bank_tracks_per_sequence;
+    uint8_t field_0x3C;
+    int8_t field_0x3D;
+    uint16_t field_0x3E;
+    uint8_t global_volumes[16];
+    struct dse_note_list_node notes[128];
+    struct dse_note_list_node* free_notes_list;
+    uint32_t field_0x654;
+    struct dse_synth* synth_list;
+    uint32_t field_0x65C;
+    struct dse_wavebank* loaded_wavebanks_list;
+    struct dse_sequence* bgm_sequences_list;
+    struct dse_sequence* se_sequences_list;
+    struct dse_se_bank* loaded_effect_banks;
+    uint8_t field_0x670[132];
+    struct dse_heap_node* heap_node_list;
+    void* heap_end;
+    int heap_size;
+    struct dse_heap_allocator heap_allocator;
+    uint8_t field_0x70C[36];
+    int16_t num_voices;
+    uint16_t active_voices_bits;
+    uint16_t start_voices_bits;
+    uint16_t deactivate_voices_bits;
+    uint16_t deallocate_voices_bits;
+    uint16_t field_0x73C;
+
+    struct dse_voice voices[16];
+
+    uint16_t field_0x1CFC;
+    uint16_t field_0x1CFE;
+    int num_active_voices;
+    int prev_nom_active_voices;
+    int ticks_until_num_active_voices_timeout;
+    uint32_t microseconds_per_driver_tick_2;
+    uint32_t field_1D10;
+    uint32_t field_1D14;
+
+    uint8_t driver_thread[192]; // Seems to be the OSThread structure from NitroSDK
+    void* driver_stack;
+
+    uint32_t field_0x1DDC;
+    struct dse_mainbank* loaded_mainbanks;
+    int mainbank_max_read_size;
+    void* mainbank_read_callback;
+    void* mainbank_read_callback_param;
+
+    uint8_t loader_thread_priority;
+    uint8_t field_0x1DF1;
+    int8_t quit_sample_loader;
+    int8_t is_sample_loader_sleeping;
+    uint8_t sample_loader_thread[192]; // Again, OSThread
+    void* sample_loader_thread_stack;
+    struct dse_wavebank* loading_bank;
+    void* loading_bank_pcm_data;
+    // There's more, but nothing labeled.
```

Comment:
Does this mean the struct is actually bigger? This comment is unclear.

Reply:
Yes. I'll update the comment to match some of the other structs with ambiguous sizes.